syntax = "proto3";

option go_package = "./pkg/io";

message PartitionState {
  double cumulative_timesteps = 1;
  int64 partition_index = 2;
  repeated double state = 3;
}